packages/contracts-bedrock/test/SystemConfig.t.sol

```diff
@@ -385,3 +385,28 @@ contract SystemConfig_CheckSequencer_TestFail is SystemConfig_Initializer {
         sysConf.checkSequencer();
     }
 }
+
+contract SystemConfig_isProposalManager_Test is SystemConfig_Initializer {
+    /// @dev Tests that `isProposalManager` returns the correct value.
+    function test_isProposalManager_succeeds() external {
+        // check if the challenger can delete an output
```

Comment:
This technically doesn't check if the challenger can delete an output, it checks that `isProposalManager` returns true. It may or may not be the case now or in the future that this function must return true to delete an output

Reply:
These comments are a bit superflous anyways, I will remove them.